kf_lib_data_ingest/etl/transform/transform.py

```diff
@@ -232,6 +236,98 @@ def handle_nulls(self, target_instances, target_schema):
 
                     instance['properties'][attr] = mapped_value
 
+    def _source_to_target_ids(self, id_cache, target_instances):
```

Comment:
Please comment on what happens here if something can only be uniquely identified by a combination of multiple values.

Reply:
Oo yes this is dealing with unique key to target id translation. I'll update the docstring and maybe even the method name